packages/manager-components/src/hooks/pci-project-provider/index.ts

```diff
@@ -3,5 +3,8 @@ import {
   useIsDiscoveryProject,
   useProject,
 } from './useProject';
+import { PublicCloudProject as Project } from './publicCloudProject.interface';
+
+export type PublicCloudProject = Project;
```

Comment:
```suggestion
export { PublicCloudProject } from './publicCloudProject.interface';
```

Reply:
I can't do this because we have a old version of prettier =>

```
[error] packages/manager-components/src/hooks/pci-project-provider/index.ts: SyntaxError: Declaration or statement expected. (7:1)
[error]    5 | } from './useProject';
[error]    6 |
[error] >  7 | export type { PublicCloudProject } from './publicCloudProject.interface';
```

Later, we can do this :)

Reply:
And if you try to remove the keyword `type` ?

Reply:
Yep and i have an issue

here https://github.com/ovh/manager/blob/master/packages/manager-components/src/components/datagrid/datagrid.component.tsx#L29 we have done similar things and a comment have been added to explain why